src/components/PlayingCard/PlayingCard.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { PlayingCard, PlayingCardProps } from '.';
+import { SUIT } from '../../types';
+import { TrophyIcon } from '../icons';
+
+export default {
+    component: PlayingCard,
+    title: 'PlayingCard',
+    args: {
+        suit: SUIT.DIAMONDS,
+        number: 7,
+    }
+}
+
+export const Example = (props: PlayingCardProps) => (
+   <PlayingCard
+        {...props}
+   />
+);
+
+export const Selected = (props: PlayingCardProps) => (
+   <PlayingCard
+        {...props}
+        selected={true}
+   />
+);
+
+export const Large = (props: PlayingCardProps) => (
+   <PlayingCard
+        {...props}
+        size={'large'}
```

Comment:
why do you use

```
prop={'string'}
```

instead of 

```
prop="string"
```

here?

Reply:
it's the same, the {} are just habit from swapping between using a variable or not etc